recipes/batman-package/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "batman-package" %}
+{% set version = "2.4.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: abc3f6b719673dc8ef856f85aad365eee3a092a76515e724571a123981fd412f
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - numpy
+    - pip
+    - python
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - batman
+
+about:
+  home: https://github.com/lkreidberg/batman
+  summary: Fast transit light curve modeling
+  license: GPL-3.0
```

Comment:
Probably needs to be SPDX

Reply:
Fixed; hopefully I also fixed the build problem on linux.